content/en/apps/reference/localizing-cht.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "CHT Localization/"
+linkTitle: "cht-localization/"
+weight: 2
+description: >
+  **Localization**: Localizing the CHT Application
+keywords: localization languages translations
+---
+
+Given that CHT apps are used around the world, the Core Framework was designed with localization in mind. The Core Framework itself is available in English, French, Hindi, Nepali, Spanish, Swahili, and Indonesian. In the `app_settings.json` file the default language for the application is set, along with a separate default language for outgoing messages that are sent via SMS. 
```

Comment:
This is a great document on tutorials, guides, explanation, and reference. https://documentation.divio.com/

You currently have this listed as reference but the task in https://github.com/medic/app-services-team/issues/33 is to create a localiation tutorial.  The good news is that you seem to have strong elements here for a great tutorial here!  

Can I challenge you to look at existing cht-docs tutorials, check out that guide for writing tutorials, and reform that content and excellent illustration to fit that pattern?